### DISCUSSION TITLE
helmwave install as alternative helm install

### DISCUSSION THREAD

**Discussion by patsevanton** [helmwave/helmwave#837] [Ideas]
Hello! Thanks for helmwave!
Share idea: helmwave install as alternative helm install.
helmwave install show progress by kubedog. That would be great.
May be new utility with kubedog for install.

**Comment by zhilyaev** [comment]
Could you share more details about what exactly you want?

**Reply by zhilyaev** [reply]
```bash
export HELMWAVE_KUBEDOG=true
helmwave up
```

**Reply by zhilyaev** [reply]
Would you do something like that and avoid the `helmwave.yml` config?
helmwave install/upgrade RELEASE CHART
